subsys/mgmt/updatehub/shell.c

```diff
@@ -59,7 +58,7 @@ static int cmd_info(const struct shell *shell, size_t argc, char **argv)
 	char *firmware_version = k_malloc(FIRMWARE_IMG_VER_STRLEN_MAX);
```

Comment:
Not for this PR but for a future PR: should be checking if either of these return NULL, otherwise the behaviour here is undefined

Reply:
Yeah, this is existing issue so we can pass over it with the PR, but please @nandojve log an issue for that or fix it (or do both) in another PR.

Reply:
Addressed at #53970